content/gsoc-2019/contents.lr

```diff
@@ -25,13 +25,23 @@ Here are short bios of each of our 2019 mentors. Please use first names or Slack
 * [Sophine Clachar](https://creativecommons.org/author/sclachar/) (`Sophine` on Slack)
 * [Timid Robot Zehta](https://creativecommons.org/author/timidcreativecommons-org/) (`Timid Robot` on Slack)
 
+### Mentor availability/ Project slots
```

Comment:
Please use consistent spacing on either side of the slash. I think that the heading for this should be clearer though, since "Mentor availability" could also refer to when we're around to chat.

Reply:
On review, I found that I did not end up putting much about mentors in this section. So, I figured that just making `Project slots` its own section would make more sense. I am unsure as to where exactly in the page this section would make the most sense (maybe after proposal review criteria?) Thoughts?
```suggestion
## Project slots
```

Reply:
Yeah, I think putting that after the proposal review criteria makes sense.

Reply:
Okie dokie, will do.